tools/chpldoc/chpldoc.cpp

```diff
@@ -74,6 +74,7 @@ bool fPrintSettingsHelp = false;
 bool fPrintChplHome = false;
 bool fPrintVersion = false;
 bool fWarnUnknownAttributeToolname = true;
+std::string CHPL_THIRD_PARTY;
```

Comment:
I don't see anywhere in your change that this gets defined. It looks like you use it on line 494 and define to a local shadow of this variable on 2283 so I'm trying to figure out when this would ever be non empty.

Reply:
this should have replaced the definition of `CHPL_THIRD_PARTY` later